proposals/lip-0038.md

```diff
@@ -102,6 +102,8 @@ Here, ceiling is an auxiliary function defined below.
 def verifyAggSig(keysList: list[bytes], aggregationBits: bytes, signature: bytes, tag: bytes, chainID: bytes, message: bytes) -> bool:
     taggedMessage = tagMessage(tag, chainID, message)
     keys = []
+    if len(aggregationBits) != ceiling(len(keysList), 8):
+      raise Exception('Length of aggregationBits bitmap is not equal to the ceiling of the length of the keys list divided by 8.')
```

Comment:
I dIdn't notice it in the first round: So far, we don't use exceptions in our BLS functions. Neither in LIP 0038 nor in 0062. So, I would rather return `false` instead. Or what do you think @shuse2 ?